docs/openstack/backend_services_deployment.md

```diff
@@ -54,6 +54,7 @@ podified OpenStack control plane services.
   AODH_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' AodhPassword:' | awk -F ': ' '{ print $2; }')
   CEILOMETER_METERING_SECRET=$(cat ~/tripleo-standalone-passwords.yaml | grep ' CeilometerMeteringSecret:' | awk -F ': ' '{ print $2; }')
   CEILOMETER_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' CeilometerPassword:' | awk -F ': ' '{ print $2; }')
+  BARBICAN_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' BarbicanPassword:' | awk -F ': ' '{ print $2; }')
```

Comment:
You are missing the barbican kek (assuming that this is for simple crypto).  This is of course assuming that the backend of the adopted system is simple_crypto.  We probably  need something about backends in the doc in the barbican section.